src/tests/unit/tests/common/components/issue-filing-button.test.tsx

```diff
@@ -98,20 +110,22 @@ describe('IssueFilingButtonTest', () => {
             needsSettingsContentRenderer,
         };
         issueFilingActionMessageCreatorMock
-            .setup(creator =>
-                creator.fileIssue(eventStub, testKey, props.issueDetailsData, toolData),
-            )
+            .setup(creator => creator.fileIssue(It.isAny(), It.isAny(), It.isAny(), It.isAny()))
             .verifiable(Times.once());
-        const wrapper = shallow<IssueFilingButton>(<IssueFilingButton {...props} />);
+        const renderResult = render(<IssueFilingButton {...props} />);
 
-        wrapper.find(DefaultButton).simulate('click', eventStub);
+        await userEvent.click(renderResult.container.querySelector('.ms-Button-label'));
```

Comment:
Another instance where it's better to *not* mock `DefaultButton` so we can use `getByRole('button')`

Reply:
resolved the comment